### PR TITLE
Multiple repository changes for GitHub actions

### PR DESCRIPTION
**Pull request by MancunianSam**
Permissions for tdr-create-db-users
Permissions for tna-custodian plus some new roles.

### CODE REVIEW

File: modules/environment-roles/root.tf [hunk 1]
```diff
@@ -185,8 +185,26 @@ resource "aws_iam_role" "custodian_deploy_role" {
   )
 }
 
+resource "aws_iam_role" "custodian_github_actions_deploy_role" {
```
Comment (TomJKing) [inline modules/environment-roles/root.tf:188]: Maybe rename this: `github_actions_custodian_deploy_role` to make it consistent with the other github actions role resources names?

File: modules/environment-roles/templates/jenkins_lambda_policy.json.tpl [hunk 2]
```diff
@@ -25,6 +25,7 @@
         "arn:aws:lambda:eu-west-2:${account_id}:function:tdr-download-files-${environment}",
         "arn:aws:lambda:eu-west-2:${account_id}:function:tdr-database-migrations-${environment}",
         "arn:aws:lambda:eu-west-2:${account_id}:function:tdr-create-keycloak-db-user-${environment}",
+        "arn:aws:lambda:eu-west-2:${account_id}:function:tdr-create-keycloak-db-user-new-${environment}",
```
Comment (TomJKing) [inline modules/environment-roles/templates/jenkins_lambda_policy.json.tpl:28]: Assume the `new` is in the name to prevent breaking changes?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of minor comments.

File: modules/environment-roles/root.tf (hunk 1 above)
Comment (MancunianSam) [reply modules/environment-roles/root.tf:188]: Yep, I've changed it here and the role attach resource as well.

File: modules/environment-roles/templates/jenkins_lambda_policy.json.tpl (hunk 2 above)
Comment (MancunianSam) [reply modules/environment-roles/templates/jenkins_lambda_policy.json.tpl:28]: Yep, it's on my list of things to do but it's referenced in quite a few places so it needs doing carefully.

**Pull request merged**